core/scheduler/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more contributor
```

Comment:
we are no longer using the short licenses. Please replace with full license here and elsewhere.